azerothcore/azerothcore-wotlk
# Fix(Core/DB): Hardcoded text - Eastern Kingdoms

##### CHANGES PROPOSED:

-  Use db data in order to allow translations with gossips and texts
-  Moved Kharan Mighthammer to DB
-  Added some overrides and some NULL to nullptr

##### TESTS PERFORMED:


Build and tested some of them.

##### HOW TO TEST THE CHANGES:


Check if all the changed gossips are working as before.
Check (if possible with locale != us) if all the changed monster yells are working as before.

##### KNOWN ISSUES AND TODO LIST:


- [ ] There are some other gossips to change.


##### Target branch(es):

Master
